python/graphstorm/run/gsgnn_lp/gsgnn_lp.py

```diff
@@ -40,11 +40,13 @@
 from graphstorm.eval import GSgnnMrrLPEvaluator
 from graphstorm.model.utils import save_embeddings
 from graphstorm.model import do_full_graph_inference
+from graphstorm.utils import rt_profiler
 
 def main(args):
     config = GSConfig(args)
 
     gs.initialize(ip_config=config.ip_config, backend=config.backend)
+    rt_profiler.init(config.profile_path, rank=gs.get_rank())
```

Comment:
Can we turn tr_profiler off?

Reply:
yes, if `profile_path` is None, `rt_profiler` is disabled.